public/js/commentForm.js

```diff
@@ -0,0 +1,44 @@
+var commentForm = (function(){
+
+    var el = {
+        $comment_input: $('#comment_input'),
+        $comment_buttons_container: $('.comment-form__button-container'),
+        $github_signin: $('#github_signin')
+    };
+
+    var bindUIElements = function() {
+        el.$comment_input.on('focus', showCommentButton);
+        el.$comment_input.on('blur', hideCommentButton);
+        el.$github_signin.on('click', handleSignin);
+    };
+
+    var init = function() {
+        bindUIElements();
+        refillPreviousComment();
+    };
+
+    var handleSignin = function (e) {
+        var comment = el.$comment_input.val();
+        localStorage.setItem('gist_comment', comment);
+    };
+
+    var showCommentButton = function () {
+        el.$comment_buttons_container.addClass('active');
+    };
+
+    var hideCommentButton = function () {
+        if(el.$comment_input.val() === "") el.$comment_buttons_container.removeClass('active');
```

Comment:
Begrudgingly yes... yes I can :+1: :smile: 

Just as an aside, I do this all over in my personal code as long as they are one liners. Not for any good reason really, probably just for terseness. Do you prefer brackets to avoid the situation where someone might try to execute multiple statements without them and cause a funky bug, or is this a PSR-2 carry over?